packages/api/src/controllers/broadcaster.js

```diff
@@ -59,4 +59,23 @@ app.get('/status', authMiddleware({}), async (req, res, next) => {
   res.json(statuses)
 })
 
+async function discoveryAuthWebhookHandler(req, res) {
+  let responseObj = {}
+  let broadcasters = []
+  try {
+    broadcasters = await req.getBroadcasters(req)
```

Comment:
The dependent on the broadcasters that the API is aware of right now results in a lot of duplication when the broadcasters use the same ETH address and price. I wonder if starting the server with `LP_PRICES` variable that accepts an array of objects of the form `{ethAddress: ..., price: }` would help.

Reply:
Fixed in 2baa3f4 , also made the necessary changes in https://github.com/livepeer/livepeer-infra/pull/390